pkg/gardenlet/apis/config/v1alpha1/defaults.go

```diff
@@ -114,6 +114,13 @@ func SetDefaults_GardenletConfiguration(obj *GardenletConfiguration) {
 		v := DefaultKubernetesLogLevel
 		obj.KubernetesLogLevel = &v
 	}
+
+	if len(obj.Server.HTTPS.BindAddress) == 0 {
+		obj.Server.HTTPS.BindAddress = "0.0.0.0"
+	}
+	if obj.Server.HTTPS.Port == 0 {
+		obj.Server.HTTPS.Port = 2020
```

Comment:
```suggestion
		obj.Server.HTTPS.Port = 2720
```
To be consistent with `charts/gardener/gardenlet/values.yaml`.